src/dotnet-interactive/CommandLine/ParseNotebookCommand.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive.Server;
+
+namespace Microsoft.DotNet.Interactive.App.CommandLine
+{
+    internal static class ParseNotebookCommand
+    {
+        public static async Task<int> Do(NotebookParserServer parserServer)
```

Comment:
FYI, you can return `Task` and skip the `return 0`. It'll product a non-zero exit code if an exception is thrown.